packages/hop-node/src/watchers/RelayWatcher.ts

```diff
@@ -0,0 +1,383 @@
+import '../moduleAlias'
+import ArbitrumBridgeWatcher from './ArbitrumBridgeWatcher'
+import BaseWatcher from './classes/BaseWatcher'
+import Logger from 'src/logger'
+import isNativeToken from 'src/utils/isNativeToken'
+import { GasCostTransactionType, TxError } from 'src/constants'
+import { L1Bridge as L1BridgeContract } from '@hop-protocol/core/contracts/L1Bridge'
+import { L2Bridge as L2BridgeContract } from '@hop-protocol/core/contracts/L2Bridge'
+import { NonceTooLowError, RelayerFeeTooLowError } from 'src/types/error'
+import { RelayableTransferRoots } from 'src/db/TransferRootsDb'
+import { Transfer, UnrelayedSentTransfer } from 'src/db/TransfersDb'
+import { isExecutionError } from 'src/utils/isExecutionError'
+import { promiseQueue } from 'src/utils/promiseQueue'
+import { providers } from 'ethers'
+import { relayTransactionBatchSize } from 'src/config'
+
+type Config = {
+  chainSlug: string
+  tokenSymbol: string
+  bridgeContract: L1BridgeContract | L2BridgeContract
+  dryMode?: boolean
+}
+
+type RelayWatchers = ArbitrumBridgeWatcher
+
+class RelayWatcher extends BaseWatcher {
+  siblingWatchers: { [chainId: string]: RelayWatcher }
+  relayWatcher: RelayWatchers
+
+  constructor (config: Config) {
+    super({
+      chainSlug: config.chainSlug,
+      tokenSymbol: config.tokenSymbol,
+      logColor: 'redBright',
+      bridgeContract: config.bridgeContract,
+      dryMode: config.dryMode
+    })
+    this.relayWatcher = new ArbitrumBridgeWatcher({
+      chainSlug: config.chainSlug,
+      tokenSymbol: this.tokenSymbol,
+      bridgeContract: config.bridgeContract,
+      dryMode: config.dryMode
+    })
+  }
+
+  async pollHandler () {
+    if (this.isL1) {
+      return
+    }
+
+    const promises: Array<Promise<any>> = []
```

Comment:
suggestion: this can be simplified a bit to

```ts
    await Promise.all([
      this.checkTransferSentToL2FromDb(),
      this.checkRelayableTransferRootsFromDb()
    ])
```